Sources/SpeziTimedWalkTest/TimedWalkTestViewModel.swift

```diff
@@ -40,15 +40,15 @@ class TimedWalkTestViewModel {
     func requestPedemoterAccess() {
         #if !targetEnvironment(simulator)
```

Comment:
Wouldn’t a better solution be to move as much code out of the if compiler directive? E.g. have additional code for simulator at the start of the method and **return** before we reach the code executed on the life device. CoreMotion compiles in the simulator it just doesn’t do anything.

In essence: don’t exclude code using compiler directives (as best as possible) only use them to add additional code special to the simulator.